marklogic-data-hub/src/main/java/com/marklogic/hub/impl/StepDefinitionManagerImpl.java

```diff
@@ -42,13 +44,16 @@
     @Autowired
     private HubConfig hubConfig;
 
+    private ArtifactService artifactService;
```

Comment:
Same thing - unless this class is intended to be bound to a single user/thread, it should not sure ArtifactService as a field.